nginx_proxy/start_nginx.sh

```diff
@@ -132,6 +145,9 @@ fi
 if [[ "${ENDPOINTS_ROLLOUT_STRATEGY}" ]]; then
   cmd+=" --rollout_strategy \"${ENDPOINTS_ROLLOUT_STRATEGY}\""
 fi
+if [[ "${ENDPOINTS_CLOUD_TRACE_AUTO_SAMPLING_FLAG}" == 'false' ]]; then
```

Comment:
Since Google control the flags from Flex passed to this script, it should be safe to allow any arbitrary flags to start_esp.

Can we support that starting now?

Reply:
Wesley talked to the GAE Admin API and Flex teams about the arbitrary flags to start_esp.

https://buganizer.corp.google.com/issues/78891740#comment5

Reply:
I deleted a comment with a Google-internal link on this thread, but no, we will not support this.